.github/workflows/test-fork-prs.yml

```
@@ -201,6 +201,36 @@ jobs:
       commit: ${{ needs.setup.outputs.commit_id }}
       repository: ${{ github.repository }}
 
+  license-checker:
+    runs-on: ubuntu-latest
+    needs: setup-cache
+    steps:
+      - name: Checkout Amplify UI
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
+        with:
+          ref: ${{ inputs.commit }}
+          repository: ${{ inputs.repository }}
```

On fork PRs, this should be:

```suggestion
    needs: [setup, setup-cache]
    steps:
      - name: Checkout Amplify UI
        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
        with:
          ref: ${{ needs.setup.outputs.commit_id }}
          repository: ${{ github.repository }}
```
`inputs` only exist in reusable workflow, and we need to get `ref` through the `setup` stage like the other checkouts in this file.

---

Oh nice, good catch